### PR TITLE
make swarm advisory more forceful

### PR DESCRIPTION
**Pull request by sfsmithcha**
Update Swarm advisory to be stronger to redirect users to "swarm mode"
Signed-off-by: Charles Smith <charles.smith@docker.com>

### CODE REVIEW

**Comment by sfsmithcha** [comment]
ping @thaJeztah , @SvenDowideit

File: data/advisories.toml [hunk 1]
```diff
@@ -13,7 +13,7 @@ rc = "The Swarm mode feature included in Docker Engine 1.12 is a release candida
 docker4mac-beta = "Docker for Mac is currently in public beta. Some functionality may change before the product becomes generally available."
 docker4win-beta = "Docker for Windows is currently in public beta. Some functionality may change before the product becomes generally available."
 
-swarm = "These documents refer to Docker Swarm cluster mangement features. For information on Docker Engine 1.12 swarm mode, see [Swarm mode overview](/engine/swarm)."
+swarm = "See [Swarm mode overview](/engine/swarm) for the orchestration features introduced in Docker Engine 1.12. Only refer to these documents for information on the standalone Swarm product."
```
Comment (thaJeztah) [inline data/advisories.toml]: really nit, but missing trailing slash after swarm; `/engine/swarm/` to prevent a redirect

**Comment by thaJeztah** [comment]
LGTM

**Comment by sfsmithcha** [comment]
ping @sanscontext

**Comment by sanscontext** [comment]
I think this is actually a little unclear. You link to and reference the new stuff, then switch to talking about the old stuff without reestablishing context - swapping these around would work better.

~"These docs refer to the old version, only use them if (x).  See the new version [here]."

**Comment by sanscontext** [comment]
Perfect. LGTM! 🚀

**Pull request merged**